app/src/molecules/JogControls/DirectionControl.tsx

```diff
@@ -8,32 +7,32 @@ import {
   SIZE_2,
   PrimaryBtn,
   Icon,
-  type IconName,
   HandleKeypress,
   ALIGN_CENTER,
 } from '@opentrons/components'
 import { ControlContainer } from './ControlContainer'
 
+import type { IconName } from '@opentrons/components'
 import type { Jog, Plane, Sign, Bearing } from './types'
 import { HORIZONTAL_PLANE, VERTICAL_PLANE } from './constants'
 
-type Control = {|
-  bearing: Bearing,
-  keyName: string,
-  shiftKey: boolean,
-  gridRow: number,
-  gridColumn: number,
-  iconName: IconName,
-  axis: 'x' | 'y' | 'z',
-  sign: Sign,
-|}
-type ControlsContents = {|
-  controls: Array<Control>,
-  title: string,
-  subtitle: string,
-|}
+interface Control {
+  bearing: Bearing
+  keyName: string
+  shiftKey: boolean
+  gridRow: number
+  gridColumn: number
+  iconName: IconName
+  axis: 'x' | 'y' | 'z'
+  sign: Sign
+}
+interface ControlsContents {
+  controls: Control[]
+  title: string
+  subtitle: string
+}
 
-const CONTROLS_CONTENTS_BY_PLANE: { [Plane]: ControlsContents } = {
+const CONTROLS_CONTENTS_BY_PLANE: { [plane in Plane]: ControlsContents } = {
```

Comment:
Do we think `Record` is more readable here?

```suggestion
const CONTROLS_CONTENTS_BY_PLANE: Record<Plane, ControlsContents> = {
```

Reply:
i think `Record` is a bit more readable